libraries/rush-lib/src/schemas/rush-project.schema.json

```diff
@@ -90,17 +90,6 @@
               "outputFolderArgumentFormat": {
                 "type": "string",
                 "description": "The command-line argument to pass to the operation to specify the output folder. The string may contain the following placeholders: {phaseName} {shardIndex}. Must end with {shardIndex}. Defaults to --shard-output-folder=\".rush/operations/{phaseName}/shards/{shardIndex}\""
-              },
-              "shardOperationSettings": {
```

Comment:
Same here - allow this to be anything so the schema validation doesn't fail for projects being updated.

Reply:
Makes sense, updated in [a438175](https://github.com/microsoft/rushstack/pull/4923/commits/a438175472486a2e95b54cf695b9c1a0d6c4fe7f) to just be a deprecate notice and support both options